src/game_battlealgorithm.cpp

```diff
@@ -925,12 +926,19 @@ bool Game_BattleAlgorithm::Normal::Execute() {
 		// FIXME: Differentiate the cases where 2k3 back attack bug applies
 		auto effect = Algo::CalcNormalAttackEffect(source, target, Game_Battler::WeaponAll, critical_hit, true, Game_Battle::GetBattleCondition(), true);
 		effect = Algo::AdjustDamageForDefend(effect, target);
-		// FIXME: Handle negative effect from attributes
+
+		// Handle negative effect from attributes
+		if (effect < 0) {
+			this->healing = true;
+			// RPG_RT bug: Negative effects have double strength
+			effect = -effect * 2;
+		}
```

Comment:
Even if it's not used in `Normal`, we should for consistency set the `negative_effect` flag here in case something later wants to rely on it.

Also this `*2` to emulate the bug is not correct. The double affect happens only for HP, but not other stats. So if you want to emulate the bug you'll need to double the hp effect but not others.

Reply:
I don't think we should emulate the animated damage number bug for better user experience in my opinion.

Reply:
I think I agree. We'll probably get "bug reports" about the effect being double, but we'll then just have to point out it's actually RPG_RT compatible!

Reply:
The double affect bug is now applied on HP only.